project1-sps/script.js

```diff
@@ -1,4 +1,78 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// Step 1 : Create mathematical equation for bot to generate a try
+  // Step 1a: Attach a randomNumber to scissors, paper or stone
+
+var botTry = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomTry;
+  var randomNumber = Math.floor(randomDecimal); // scissors, paper, stone
+
+  if (randomNumber == 0) {
+    return "scissors";
+  } 
+  
+  if (randomNumber == 1) {
+     return "paper";
+  }
+  
+  if (randomNumber == 2) {
+    return "stone";
 };
```

Comment:
Nice helper function! You may want to rename it to something more meaningful like `generateRandomMove()`